packages/tablev2/style.ts

```diff
@@ -0,0 +1,85 @@
+import * as dt from "../design-tokens/build/js/designTokens";
+import { css, cx } from "emotion";
+import { pseudoElTriangle } from "../shared/styles/styleUtils";
+
+export const headerCss = css`
+  box-sizing: border-box;
+  border-top: 1px solid ${dt.themeBorderHeavy};
+  border-bottom: 1px solid ${dt.themeBorderHeavy};
+`;
+
+export const table = (
+  gridTemplateFragments: string[],
+  firstColSticky: boolean = false
+) => css`
+  display: grid;
+  grid-template-columns: ${gridTemplateFragments.join(" ")};
+  position: relative;
+  overflow: scroll;
+  ${firstColSticky && stickyFirstCol(gridTemplateFragments.length)}
```

Comment:
Is there a reason you chose to apply the styles this way instead of using emotion's `cx` function in the table element's `className` prop?

Then, you could do:
```jsx
<div
  className={cx(
    tableStyle,
    className,
    {[stickyFirstCol(state.columns.length)] : config.firstColSticky}
  )
} {...divProps} />
```

Reply:
no! your suggestion looks so much better.